src/android/src/com/pylonproducts/wifiwizard/WifiWizard.java

```diff
@@ -361,17 +361,58 @@ private boolean listNetworks(CallbackContext callbackContext) {
        *    of the scanned networks.
        *
        *    @param    callbackContext        A Cordova callback context
+       *    @param    data                   JSONArray with [0] == JSONObject
        *    @return    true
        */
-    private boolean getScanResults(CallbackContext callbackContext) {
+    private boolean getScanResults(CallbackContext callbackContext, JSONArray data) {
         List<ScanResult> scanResults = wifiManager.getScanResults();
 
         JSONArray returnList = new JSONArray();
 
+        Integer numLevels = null;
+
+        if (!data.isNull(0)) {
+            try {
+                JSONObject options = data.getJSONObject(0);
+
+                if (options.has("numLevels")) {
+                    Integer levels = options.optInt("numLevels");
+
+                    if (levels > 0) {
+                        numLevels = levels;
+                    } else if (options.optBoolean("numLevels", false)) {
+                        // use previous default for {numLevels: true}
+                        numLevels = 5;
+                    }
+                }
+            } catch (JSONException e) {
+                e.printStackTrace();
+            }
+        }
+
         for (ScanResult scan : scanResults) {
+            /*
+             * @todo - breaking change, remove this notice when tidying new release and explain changes, e.g.:
```

Comment:
FYI: I also included this @todo in case you wanted to include more details on the breaking change in the changelog.